Makefile

```diff
@@ -300,6 +300,14 @@ $(ISO_BUILD_IMAGE): deploy/iso/minikube-iso/Dockerfile
 	@echo ""
 	@echo "$(@) successfully built"
 
+storage-provisioner-main: cmd/storage-provisioner/main.go
```

Comment:
This should technically depend on all the storage-provisioner files.

Reply:
This is not great, but you'll need do add something that computes the dependencies,

`STORAGE_PROVISIONER_FILES :=  GOPATH=$(GOPATH) go list  -f '{{join .Deps "\n"}}' k8s.io/minikube/cmd/storage-provisioner | grep k8s.io | GOPATH=$(GOPATH) xargs go list -f '{{ range $$file := .GoFiles }} {{$$.Dir}}/{{$$file}}{{"\n"}}{{end}}'`